test/unit/math/mix/fun/log_mix_part3_test.cpp

```diff
@@ -0,0 +1,59 @@
+#include <test/unit/math/test_ad.hpp>
+#include <limits>
+#include <vector>
+
+std::vector<Eigen::VectorXd> to_vectors(
+    const std::vector<std::vector<double>>& xs) {
+  std::vector<Eigen::VectorXd> ys;
+  for (const auto& x : xs)
+    ys.push_back(stan::test::to_vector(x));
+  return ys;
+}
+
+std::vector<Eigen::RowVectorXd> to_row_vectors(
+    const std::vector<std::vector<double>>& xs) {
+  std::vector<Eigen::RowVectorXd> ys;
+  for (const auto& x : xs)
+    ys.push_back(stan::test::to_row_vector(x));
+  return ys;
+}
+
+void expect_log_mix(const std::vector<double>& p,
+                    const std::vector<std::vector<double>>& ds) {
+  auto f
+      = [](const auto& x, const auto& y) { return stan::math::log_mix(x, y); };
+  stan::test::ad_tolerances tols;
+  tols.hessian_hessian_ = 1e-2;
+  tols.hessian_fvar_hessian_ = 1e-2;
+
+  auto p_v = stan::test::to_vector(p);
+  auto p_rv = stan::test::to_row_vector(p);
+
+  auto ds_v = to_vectors(ds);
+  auto ds_rv = to_row_vectors(ds);
+
+  stan::test::expect_ad(tols, f, p_rv, ds);
+  stan::test::expect_ad(tols, f, p_rv, ds_v);
+  stan::test::expect_ad(tols, f, p_rv, ds_rv);
```

Comment:
Also need to add ```expect_ad``` with ```p_v``` and ```p```  inputs

Reply:
These tests are in part2. Although I am not sure anymore if that split was necessary. Meanwhile I also found other tests that seem to big for my PC, but work fine on Jenkins (I still can run them if I put `-O1` into `make/local`). I will try to revert this and see if it passes on Jenkins.